admin-commands/reaction-roles.md

```diff
@@ -5,82 +5,66 @@
 
-This command starts a guided prompt to assign emojis and roles to a group. First, you will be asked to send a message with the role (can be a name, mention or ID), then, you'll be asked to add a reaction to confirm. You can end the guided configuration at any time.
-
-{% hint style="warning" %}
-Using this command on a group that has been previously set will delete all existing emojis and roles and make you start again. If you don't want to overwrite any previously set option, add the `no-overwrite` argument.
-{% endhint %}
+![](<../.gitbook/assets/ReactionRoleSetup2.png>)
 
-#### Apply
-
-Applies the selected group to a given message. The arguments are `group`, which is the name of the group you want to assign and `message`, corresponding to a message ID or link to the message you want the group to be applied to.
-
-{% hint style="info" %}
-More than one group can be applied to a same message at any time. The same group can be applied to a maximum of 5 different messages.
-{% endhint %}
+The required arguments for this option are `group`, `emoji`, and `role`. `Group` is the group that you want to apply pairs to and `emoji` is the emoji you want the user to react with to toggle the `role`.
 
-#### Unapply
+#### Delete
 
-Has the opposite effect of `apply`, ergo, it will remove the group from the specified message. Arguments are the same as the apply option.
-
-#### Delete-group
-
-This deletes a given group as required argument.
+This deletes a given group (`group` is the required argument.).
 
 {% hint style="danger" %}
 This action is irreversible. You will be allowed to create a new group with the same name as a deleted group.
 {% endhint %}
 
-#### Group-info
+#### Info
 
-Takes a group name as required argument, and shows some informations, such as the emoji/role couples and settings for the group.
+Takes a group name as required argument, and shows some information, such as the emoji/role couples and settings for the group.
 
 {% hint style="info" %}
 You can edit group options with the `config` command
 {% endhint %}
 
-#### Listgroups
+#### List
 
-This option lists all the available groups by name and n° of roles set. No arguments needed.
+This option lists all the available groups by name and amount of roles set. No arguments required.
 
 #### Autorepair
 
-This removes any deleted emoji/role from the group passed as argument, cleaning things up. It will not search for deleted messages.
+This removes any deleted emoji/role from the group (the only required argument), cleaning things up. It will not search for deleted messages.
+
+{% hint style="info" %}
+There is an optional argument (`disconnect-messages`) that removes the group from all messages it is applied to. This is meant to be used if a group reaches the maximum of 5 messages, because there is (currently) no way to remove a group from an individual message (not even by deleting the message). 
```

Comment:
> because there is (currently) no way to remove a group from an individual message (not even by deleting the message)

Is incorrect - it's possible to remove a message from a group with the right click command. What I meant to say is that deleting the message does not automatically remove it from the list of deleted groups, and a build-up of 5 messages would make the group unusable. `disconnect-messages` is a last resort move for this scenario and internally it works by emptying the group's list